Documentation/ceph-dashboard-monitoring.md

```diff
@@ -0,0 +1,291 @@
+
+Next step is to instruct the Prometheus server about how to scrape metrics from these pods, execute:
+
+```console
+$ kubectl create secret generic additional-scrape-configs --from-file=prometheus-additional.yaml -oyaml -n rook-ceph > additional-scrape-configs.yaml
+$ kubectl create -f additional-scrape-configs.yaml
+$ kubectl patch prometheus rook-prometheus -n rook-ceph --type merge --patch "$(cat scrape-config-patch.yaml)
+```
+
+Finally, just verify that the metrics coming from the cluster nodes are present in the Prometheus server.
+
+```console
+$ echo "http://$(kubectl -n rook-ceph -o jsonpath={.status.hostIP} get pod prometheus-rook-prometheus-0):30900/targets"
+http://192.168.122.70:30900/targets
+```
+
+Type the URL returned in your internet browser, you should see something like:
+
+![Grafana home page](media/prometheus_targets.png)
+
+# Prometheus alert manager
+
+In order to enable the complete set of alerting features provided by the Ceph dashboard, a Prometheus Alert Manager must be accessible.
+
+Execute:
+```console
+# kubectl create -f alert-manager.yaml
+configmap/alertmanager-config created
+service/alertmanager created
+deployment.apps/alertmanager created
+```
+
+You should be able now to connect with the alert manager using the following URL:
+
+```console
+$ echo "http://$(kubectl get pod -l app=alertmanager -n rook-ceph -o jsonpath={.items[0].status.hostIP}):$(kubectl get service alertmanager -n rook-ceph  -o jsonpath='{.spec.ports[0].nodePort}')"
+http://192.168.122.70:31000
+```
+You will get something like:
+
+![Alert manager home page](media/alert-manager.png)
+
+Once deployed the Prometheus Alert Manager is needed to modify the Prometheus server configuration to point the new Alert manager.
+
+Execute:
+```console
+$ kubectl patch prometheus rook-prometheus -n rook-ceph --type merge --patch "$(cat alert-manager-patch.yaml)"
+```
+
+Verify that the Prometheus Configuration is set, accessing the Prometehus server UI, go to "Status-configuration" you should see the reference to the alert manager in the configuration file:
+
+```console
+alertmanagers:
+- kubernetes_sd_configs:
+      - role: endpoints
+        namespaces:
+        names:
+            - rook-ceph
+```
+
+The last step in the Prometheus alert manager configuration is to update a couple of Ceph dashboard settings:
+
+- Get the alert manager URL:
+```console
+$ echo "http://$(kubectl get pod -l app=alertmanager -n rook-ceph -o jsonpath={.items[0].status.hostIP}):$(kubectl get service alertmanager -n rook-ceph  -o jsonpath='{.spec.ports[0].nodePort}')"
+http://192.168.122.70:31000
+```
+- Open a Ceph toolbox and configure the alert manager api host:
+
+```console
+[root@rook-ceph-tools-78cdfd976c-8xvgn /]# ceph dashboard set-alertmanager-api-host 'http://192.168.122.70:31000'
+Option ALERTMANAGER_API_HOST updated
+[root@rook-ceph-tools-78cdfd976c-8xvgn /]# ceph dashboard set-alertmanager-api-ssl-verify False
+Option ALERTMANAGER_API_SSL_VERIFY updated
```

Comment:
same here link to the toolbox and keep each command different and their outpur
```suggestion
- Open a Ceph toolbox and configure the alert manager api host:

```console
ceph dashboard set-alertmanager-api-host 'http://192.168.122.70:31000'
Option ALERTMANAGER_API_HOST updated
```
```console
ceph dashboard set-alertmanager-api-ssl-verify False
Option ALERTMANAGER_API_SSL_VERIFY updated
```